configs/openmrs_config/roles/cambodia_roles.csv

```diff
@@ -1,2 +1,4 @@
+8c26a98b-fa03-4c37-b788-6d1e537d4071,NCD care provider in Health Center,NCD care provider in Health Center,,View NCD Screening encounters; Edit NCD Screening encounters;View NCD Medical History encounters;View NCD Consultation encounters,
+c65ad447-bbe8-4917-abc6-30617dbac994,NCD care provider in Hospital,NCD care provider in Hospital,,View NCD Medical History encounters;Edit NCD Medical History encounters;View NCD Consultation encounters;Edit NCD Consultation encounters;Vew NCD Screening encounters,
```

Comment:
@kdaud, according to the ticket description, you should use the `Inherited roles` field to inherit from the required roles instead of adding their corresponding individual privileges.